documentation/docs/release-notes/3.1.0.md

```diff
@@ -0,0 +1,170 @@
+# Percona Monitoring and Management 3.1.0 
+
+| **Release date** | March 27th, 2025                                                                                  |
+| ----------------- | :---------------------------------------------------------------------------------------------- |
+| **Installation** | [Installing Percona Monitoring and Management](../quickstart/quickstart.md) |
+| **Upgrade**| [Migrate PMM 2 to PMM 3](../pmm-upgrade/migrating_from_pmm_2.md)
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+PMM empowers you to: 
+
+- monitor the health and performance of your database systems
+- identify patterns and trends in database behavior
+- troubleshoot issues effectively
+- execute database management tasks seamlessly, whether your databases are on-premises or in the cloud
+
+## Release summary
+
+PMM 3.1.0 enhances MongoDB monitoring with improved query analytics, better visualization, and streamlined troubleshooting. This release also refines role-based access for advisors and alerts, along with various usability improvements and bug fixes.
+
+## Release highlights
+
+### Enhanced MongoDB monitoring
+We've improved MongoDB monitoring with detailed query metrics, better dashboard visualization, and new capabilities to quickly identify performance bottlenecks.
+
+#### Extended metrics and labels for MongoDB Query Analytics (QAN)
+
+The [QAN dashboard](../use/qan/index.md) includes new metrics and filters that highlight inefficient execution paths and offer detailed lock and storage metrics. With the improved filters, you can quickly pinpoint problematic queries based on application, client, user, or plan summary:
+
+##### New metrics
+In the **Query Details** section of the QAN dashboard you'll now find these additional metrics:
+
+=== "Query execution metrics"
+    New metrics for identifying inefficient query patterns:
+
+    - **Docs Examined**: the number of documents scanned. High values may indicate missing or inefficient indexes.
+    - **Keys Examined**: the number of index keys scanned. Compare this with **Docs Examined** to assess index efficiency.
+    - **Docs Returned**: the number of documents returned to the PMM Client. A high **Docs Examined** to **Docs Returned** ratio may indicate inefficiencies that need optimization. 
```

Comment:
```suggestion
    - **Docs Returned**: the number of documents returned. A high **Docs Examined** to **Docs Returned** ratio may indicate inefficiencies that need optimization. 
```
I don't believe this has anything to do with being returned to the PMM Client... but feel free to ignore if it does and I just don't understand.

Reply:
Yeah, you are right, it's a number of documents returned to the client queried them.